tests/unit/filter/test_lucene_filter.py

```
@@ -497,3 +497,18 @@ def test_new_lucene_compliance_single_escape(self):
         lucene_filter = LuceneFilter.create("regex_key_one:/\/.*value.*/")
 
         assert lucene_filter == RegExFilterExpression(["regex_key_one"], "\/.*value.*")
+
+    def test_new_lucene_compliance_parentheses(self):
+        lucene_filter = LuceneFilter.create("regex_key_one:(/.*value.*/)")
+
+        assert lucene_filter == RegExFilterExpression(["regex_key_one"], ".*value.*")
+
+    def test_creates_lucene_compliance_filter_parentheses_or(self):
```

Additionally, a test for `'regex_key: (/.*value_one.*/ OR /.*value_two.*/)'` would be good.

Furthermore, `'regex_key_conflict: (/.*value_one.*/ OR "/.*value_two.*/")'` should probably raise an exception, since there is a contradiction for how to parse `regex_key_conflict`. Therefore, a test for this would be good as well.

---

Added `'regex_key: (/.*value_one.*/ OR /.*value_two.*/)'`. 
Altered `'regex_key_one:(/.*value.*./ OR key_two: "/.*value.*/")' ` to `'regex_key_one: (/.*value_one.*/ OR /.*value_two.*/)'`. 

Are more nested parentheses structures necessary?

---

> Added `'regex_key: (/.*value_one.*/ OR /.*value_two.*/)'`. Altered `'regex_key_one:(/.*value.*./ OR key_two: "/.*value.*/")' ` to `'regex_key_one: (/.*value_one.*/ OR /.*value_two.*/)'`.
> 
> Are more nested parentheses structures necessary?

Thank you! Yes, parentheses can reduce the size of a filter of by a lot, which makes it both easier to write and read.

---

> ... no conflict raised here:
> 
> `lucene_filter = LuceneFilter.create('regex_key_one: (/.*value_one.*/ OR "/.*value_two.*/")') assert lucene_filter == Or( RegExFilterExpression(["regex_key_one"], ".*value_one.*"), StringFilterExpression(["regex_key_one"], "/.*value_two.*/"), )`

Sorry, it was a mistake. I have edited it out a while ago.